conf/messages.en

```diff
@@ -882,24 +866,25 @@ title.turkey=Turkey
 title.turkmenistan=Turkmenistan
 title.turks_and_caicos_islands=Turks and Caicos Islands
 title.tuvalu=Tuvalu
+title.ussr=USSR
 title.uganda=Uganda
 title.ukraine=Ukraine
 title.united_arab_emirates=United Arab Emirates (Abu Dhabi, Ajman, Dubai, Fujaairah, Ras al Khaimah, Sharjah and Umm al Qaiwain)
 title.umm_al_qaiwain=Umm al Qaiwain
```

Comment:
this is "Umm al-Quwain" in the spreadsheet

Reply:
👍